internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -279,13 +281,13 @@ func (r *BMACReconciler) Reconcile(origCtx context.Context, req ctrl.Request) (c
 	}
 
 	if result.Stop(ctx) {
-		log.Debugf("Stopping BMAC reconcile after ensureMCSCert")
+		log.Infof("Stopping BMAC reconcile after ensureMCSCert")
 		return result.Result()
 	}
 
 	// After the agent has started installation, Ironic should not manage the host.
 	// Adding the detached annotation to the BMH stops Ironic from managing it.
-	result = r.addBMHDetachedAnnotationIfAgentHasStartedInstallation(ctx, bmh, agent)
+	result = r.addBMHDetachedAnnotationIfAgentHasStartedInstallation(log, ctx, bmh, agent)
```

Comment:
ctx usually comes first, please keep it as a first param

Reply:
+1